lib/utils.pm

```diff
@@ -391,7 +391,17 @@ sub zypper_call {
     return $ret;
 }
 
+# creae btrfs subvolume for aarch64.
+# ref:bsc#1122591
+sub create_btrfs_subvolume {
+    return unless check_var('ARCH', 'aarch64');
+    assert_script_run("mv /boot/grub2/arm64-efi /boot/grub2/arm64-efi.bk");
+    assert_script_run("btrfs subvolume create /boot/grub2/arm64-efi");
+    assert_script_run("cp -r /boot/grub2/arm64-efi.bk/* /boot/grub2/arm64-efi/");
+}
+
 sub fully_patch_system {
+    create_btrfs_subvolume();
```

Comment:
The workaround shouldn't belong to fully patch for system, so it is not suitable to invoke the function here. I suggest it to be invoked in patch_sle.pm after 'install_salt_packages()'. WDYT?